src/views/Lottery/components/ClaimPrizesModal/ClaimPrizesInner.tsx

```diff
@@ -25,6 +25,11 @@ const ClaimInnerContainer: React.FC<ClaimInnerProps> = ({ onSuccess, roundsToCla
   const { toastSuccess, toastError } = useToast()
   const [activeClaimIndex, setActiveClaimIndex] = useState(0)
   const [pendingTx, setPendingTx] = useState(false)
+  const [pendingBatchClaims, setPendingBatchClaims] = useState(
+    Math.ceil(
+      roundsToClaim[activeClaimIndex].ticketsWithUnclaimedRewards.length / maxNumberTicketsPerBuyOrClaim.toNumber(),
```

Comment:
Slight refactor to the data shown when a user needs multiple ticket claims in one round